history/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/query/HistoryQueryColumnName.java

```diff
@@ -26,16 +26,13 @@
   TASK_CLASSIFICATION_KEY("task_classification_key"),
   TASK_CLASSIFICATION_CATEGORY("task_classification_category"),
   ATTACHMENT_CLASSIFICATION_KEY("attachment_classification_key"),
-  COMMENT("comment"),
   OLD_VALUE("old_value"),
   NEW_VALUE("new_value"),
   CUSTOM_1("custom_1"),
   CUSTOM_2("custom_2"),
   CUSTOM_3("custom_3"),
   CUSTOM_4("custom_4"),
-  OLD_DATA("old_data"),
-  NEW_DATA("new_data"),
-  TYPE("type");
+  TYPE("event_Type");
```

Comment:
'event_type'?

Reply:
I removed this since there is no such field "type" in the database, only "event_type" but now I saw that "event_type" is already defined some lines before that